tests/utils/test-data-generator.ts

```diff
@@ -473,18 +494,27 @@ export class TestDataGenerator {
   /**
    * Generates a PermissionsRequest message for testing.
    */
-  public static async generatePermissionsRequest(): Promise<{ message: BaseMessage }> {
-    const { privateJwk } = await ed25519.generateKeyPair();
-    const permissionRequest = await PermissionsRequest.create({
-      dateCreated                 : getCurrentTimeInHighPrecision(),
-      description                 : 'drugs',
-      grantedBy                   : 'did:jank:bob',
-      grantedTo                   : 'did:jank:alice',
-      scope                       : { method: 'RecordsWrite' },
-      authorizationSignatureInput : { privateJwk: privateJwk, protectedHeader: { alg: privateJwk.alg as string, kid: 'whatev' } }
+  public static async generatePermissionsRequest(input?: GeneratePermissionsRequestInput): Promise<GeneratePermissionsRequestOutput> {
+    const author = input?.author ?? await TestDataGenerator.generatePersona();
+    const permissionsRequest = await PermissionsRequest.create({
+      dateCreated : getCurrentTimeInHighPrecision(),
+      description : input?.description ?? 'drugs',
```

Comment:
If you took the other comment, then we can get rid of the drug reference here. 🤣 
```suggestion
      description : input?.description,
```